src/models/ContentItem.ts

```diff
@@ -0,0 +1,10 @@
+export class ContentItem {
+  name: string;
+  /**
+   * Field is excluded from the response when a locale is set
+   */
+  id: string;
+  localizedNames: string;
```

Comment:
Is this a string, or should we have `LocalizedNames` as an `enum`?

Reply:
This has been updated to reflect the changes referenced.